replication_stats.go

```diff
@@ -13,10 +25,26 @@ type ReplicationStats struct {
 	numAttachmentsTransferred  uint64
 	attachmentBytesTransferred uint64
 	docsCheckedSent            uint64
+	active                     atomicBool
 	endLastSeq                 interface{}
 	endLastSeqLock             sync.RWMutex
 }
 
+// String returns the given ReplicationStats as JSON to satisfy the expvar.Var interface
+func (rs *ReplicationStats) String() string {
+	return fmt.Sprintf(`{"%s":%t,"%s":%d,"%s":%d,"%s":%d,"%s":%d,"%s":%d}`,
```

Comment:
I'm not too happy about this, but unsure how to nicely marshal the struct into JSON while retaining the atomic loads.

@adamcfraser any ideas?